jobs/pipelines/powervm/custom-powervm-jobs/custom-ocp4.x-powervm-p8-upgrade/Jenkinsfile

```diff
@@ -0,0 +1,345 @@
+                           default:
+                               return['Could not get Env'];
+                        }
+                    '''
+                ]
+            ]
+        ],
+        [$class: 'ChoiceParameter',
+            choiceType: 'PT_SINGLE_SELECT',
+            description: 'Select the Configuration',
+            filterLength: 1,
+            filterable: false,
+            name: 'Config',
+            script: [
+                $class: 'GroovyScript',
+                fallbackScript: [
+                    classpath: [],
+                    sandbox: false,
+                    script:
+                        'return[\'Could not get Env\']'
+                ],
+                script: [
+                    classpath: [],
+                    sandbox: false,
+                    script:
+                        'return["min:selected","max"]'
+                ]
+            ]
+        ]
+    ])
+])
+pipeline {
+    agent {
+        docker {
+            image 'quay.io/powercloud/inbound-agent:4.6-1.2'
+            args '-v /etc/resolv.conf:/etc/resolv.conf'
+            label 'jump-vpc-x86_64'
+        }
+    }
+    parameters {
+        string(defaultValue: '', description: 'Current Build(quay image or build number)', name: 'CurrentBuild')
+        string(defaultValue: '', description: 'Upgrade Build(quay image or build number)', name: 'UpgradeBuild')
+        string(defaultValue: '0', description: 'Enter time(in Minutes) to retain the cluster', name: 'KeepFor')
+        booleanParam(defaultValue: false, name: 'Rune2e', description: 'Select if you want to run e2e tests. E2e runs after cluster upgrade')
+    }
+    environment {
+        //users and credentials. All must be defined in Jenkins Credentials
+        GITHUB_USER = credentials('GITHUB_USER')
+        GITHUB_TOKEN = credentials('GITHUB_TOKEN')
+        OS_USERNAME = credentials('OS_USERNAME')
+        OS_PASSWORD = credentials('OS_PASSWORD')
+        DOCKER_USER = credentials('DOCKER_USER')
+        ARTIFACTORY_USER = credentials('ARTIFACTORY_USER')
+        ARTIFACTORY_TOKEN = credentials('ARTIFACTORY_TOKEN')
+        REDHAT_USERNAME = credentials('REDHAT_USERNAME')
+        REDHAT_PASSWORD = credentials('REDHAT_PASSWORD')
+        PULL_SECRET = credentials('PULL_SECRET')
+
+        //Parameters
+        OCP_RELEASE="${params.Release}"
+        UPGRADE_RELEASE="${params.UpgradeRelease}"
+        CONFIG="${params.Config}"
+        CURRENT_BUILD="${params.CurrentBuild}"
+        UPGRADE_BUILD="${params.UpgradeBuild}"
+        TIMEOUT = "${params.KeepFor}"
+        ENABLE_E2E_TEST="${params.Rune2e}"
+
+        //Env constants
+        HARDWARE_CHOSE = "P8"
+        AVAILABILITY_ZONE = "zstream"
+        TARGET = "deploy-openshift4-powervc"
+        TEMPLATE_FILE = ".${TARGET}.tfvars.template"
+        POWERVS = false
+        WAIT_FOR_DEBUG = "0"
+        REDHAT_RELEASE = "8.3"
+        ENABLE_SCALE_TEST="false"
```

Comment:
I think we need to add SCRIPT_DEPLOYMENT = false to all the powervm ocp Jenkinsfile

Reply:
@sudeeshjohn Added that in jenkins file